### ISSUE TITLE
Not working on Windows VPS. Listed all steps . What am I doing wrong ?

### ISSUE THREAD

**Issue by picusc** [davidmann4/ogar-feeder-bot#395]
Here is what I did:

1) Downloaded the zip(this repo) to the VPS and extracted it. Installed NodeJS on the VPS. Changed directory to the repo from CMD and did **npm install** and **npm install socket.io-client** respectively. 

2) Opened config,js and changed Line 15 : `config.feederServer = "ws://104.236.100.252:8081"` to `config.feederServer = "ws://My VPS's local IP:8081"`

3) Installed Tampermonkey in my local PC and installed **agar-feeder-bot.user** file.

4) Found out the UUID of my local PC after installing the Tampermonkey script. Went back to the VPS and I put the same UUID(from my local PC) in line 18 of config.js : **config.client_uuid = "c3dd"**;

5) Went to Firewall settings of my VPS and portforwarded Port 8081. Here is an image : [http://i.imgur.com/xS94c62.png](http://i.imgur.com/xS94c62.png)

6) Got fresh proxy from [HidemyAss Proxy](https://www.hidemyass.com/proxy) and put them in proxy.txt on the VPS.

7) Saved everything and ran **node feeder.js** on the VPS and went to agario on my local PC and there are no bots at all !!!!

Please help me. What mistake did I do? Why is it not working ?

**Comment by Nick8fr32** [comment]
did u change the ip in tampermoney to ur vps 
**line 84 =     var socket = io.connect('YOUR_VPS');**

**Comment by picusc** [comment]
Yes, I did. But that line is on Line 26 in my Tampermonkey script.

**Comment by ElieeCh** [comment]
All you have to do is make a .bat file that opens the server for you and also one that starts the _feeder.js_
One more that starts both of the .bat files and it should work.
Hope this helps :)

**Comment by philw1508** [comment]
Nope the only way on Windows vps is to port forward on the router

**Comment by picusc** [comment]
Portforward what port on the router? I have already portforwarded port 8081(on both my local PC and VPS) but I dont understand why that's necessary because both feeder.js and server.js are running in the VPS, not on my local PC.

**Comment by ghost** [comment]
well can we make an button on scripte to restart feeder.js

**Comment by VoOoLoX** [comment]
Try adding port rule into both "Inbound rules" and "Outbound rules" also try setting "Remote port" option to 8081 as well

**Comment by ghost** [comment]
is ur vps works now ? if it's tell me how ur bots run
